src/panels/profile/ha-profile-section-general.ts

```diff
@@ -128,6 +144,26 @@ class HaProfileSectionGeneral extends LitElement {
               .narrow=${this.narrow}
               .hass=${this.hass}
             ></ha-pick-first-weekday-row>
+            ${this._browserThemeSettings || this._browserThemeActivated
+              ? html`
+                  <ha-settings-row .narrow=${this.narrow}>
+                    <span slot="heading">
+                      ${this.hass.localize("ui.panel.profile.themes.header")}
+                    </span>
+                    <span class="theme-warning" slot="description">
+                      <ha-svg-icon .path=${mdiAlertCircleOutline}></ha-svg-icon>
+                      ${this.hass.localize(
+                        "ui.panel.profile.themes.device.user_theme_info"
+                      )}
```

Comment:
We should allow changing the user theme even if a device theme is configured.

Reply:
:+1: I'll show the warning and the user theme selector